cscs-checks/mch/openacc_cuda_mpi_cppstd.py

```diff
@@ -0,0 +1,57 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+@rfm.parameterized_test([True], [False])
+class OpenaccCudaMpiNoMPICppstd(rfm.RegressionTest):
```

Comment:
The name is weird! Why not just `OpenaccCudaCpp`, which is in accordance with the issue as well?

Reply:
@teojgo Can you change the name of this class? Other than this, this is ready to be merged.

Reply:
I don't think the automatically generated name will be very meaningful for this one: `OpenaccCudaCpp_False` and `OpenaccCudaCpp_True`. So it should be set manually.